scripts/main.js

```diff
@@ -31,6 +32,65 @@ module.exports = (robot) => {
             }
         }
     })
+    
+    robot.hear(/bot create milestone|bot close milestone/i, (res) => {
+        var message = res.message.text
+        message = message.split(' ')
+        if(message.length === 7 && message[0] === "bot" && message[1] === "create" && message[2] === "milestone") {
+          var repos = message[6]
+          repos = repos.split('/')
+          var title = message[3]+" "+message[4]+" "+message[5]
+          const myDate = moment(title, 'DD-MMM-YYYY').toDate()
+          if(myDate == "Invalid Date") {
+            res.reply("Title format is invalid")
```

Comment:
If invalid you should create milestone without start and end date

Reply:
@zelima As i mentioned above, I will implement part for `icebox` and `backlog`